server/base/src/main/java/org/apache/accumulo/server/metadata/TabletMutatorBase.java

```diff
@@ -95,6 +95,21 @@ protected TabletMutatorBase(ServerContext ctx, KeyExtent extent) {
     return this;
   }
 
+  @Override
+  public Ample.TabletMutator putFlushId(long flushId) {
+    Preconditions.checkState(updatesEnabled, "Cannot make updates after calling mutate.");
+    TabletsSection.ServerColumnFamily.FLUSH_COLUMN.put(mutation,
+        new Value(("" + flushId).getBytes()));
+    return this;
+  }
+
+  @Override
+  public Ample.TabletMutator putTime(String time) {
```

Comment:
I thought of #1210 while looking at this.  Using String is fine for this PR as adding the type would be a much bigger and unrelated change.

Reply:
Yes,  a time object would be good.  We can update later.